frontend/components/book-table-page/timeUtils.ts

```diff
@@ -0,0 +1,12 @@
+// timeUtils.ts
+export const isTimeWithinRange = (time: string, start: string, end: string) => {
```

Comment:
I'm not sure if you want to pass in time as string. This timeUtils should use some kinds of library (Datetime for example) to handle timestamps instead of using strings to do calculation here.